crates/starknet-types-core/src/curve/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod affine_point;
+pub mod curve_errors;
+pub mod projective_point;
```

Comment:
Not sure how we want to expose the module tree, but I think simply exporting those modules directly would be bettre API-wise.

```rust
mod affine_point;
mod curve_errors;
mod projective_point;

pub use self::affine_point::*;
pub use self::projective_point::*;
pub use self::curve_point::*;
```

Something like that.